pkg/types/gcp/validation/machinepool.go

```diff
@@ -33,6 +34,15 @@ func ValidateMachinePool(platform *gcp.Platform, p *gcp.MachinePool, fldPath *fi
 		}
 	}
 
+	for i, tag := range p.Tags {
+		if tag == "" {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("tags").Index(i), tag, fmt.Sprintf("Tag can not be empty")))
+		} else if !regexp.MustCompile(`^[a-z0-9-]*$`).MatchString(tag) {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("tags").Index(i), tag, fmt.Sprintf("Tag can only contain lowercase letters, numbers, and dashes")))
+		} else if len(tag) > 63 {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("tags").Index(i), tag, fmt.Sprintf("Maximum number of characters is 63")))
```

Comment:
error strings shouldn't be capitalized, cf: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

(although note that Log lines are capitalized)